pandas/_libs/tslibs/strptime.pyx

```diff
@@ -93,6 +114,7 @@ def array_strptime(
     exact : matches must be exact if True, search if False
     errors : string specifying error handling, {'raise', 'ignore', 'coerce'}
     """
+    from pandas._libs.tslibs.parsing import format_is_iso
```

Comment:
no way to avoid this?

Reply:
yup, can move it to `strptime` - thanks!